src/lib/__tests__/validators.ts

```diff
@@ -59,3 +59,37 @@ describe('validateEmail', () => {
     });
   });
 });
+
+describe('Validate The valid displayName to be true', () => {
+  [
+    'Ähtö',
+    'Aki',
+    'Äteritsiputeritsipuolilautatsijänkä',
+    'Veera',
+    'Ivan',
+    'eppu',
+    'feelix',
+    'pyry',
```

Comment:
Probably unnecessary to have all of these when the test-case is basically the same. Maybe add a name with a number, or an allowed special character in the mix.